gemfiles/rails_5_2.gemfile

```diff
@@ -2,15 +2,16 @@
 
 source "https://rubygems.org"
 
-gem "activerecord-jdbcsqlite3-adapter", platform: :jruby
-gem "bcrypt", "~> 3.1", require: false
-gem "rails", "~> 5.2.2", ">= 5.2.2.1"
+gem "rails", "~> 5.2.0"
 gem "rspec-core", git: "https://github.com/rspec/rspec-core.git"
 gem "rspec-expectations", git: "https://github.com/rspec/rspec-expectations.git"
 gem "rspec-mocks", git: "https://github.com/rspec/rspec-mocks.git"
 gem "rspec-rails", branch: "4-0-dev", git: "https://github.com/rspec/rspec-rails.git"
 gem "rspec-support", git: "https://github.com/rspec/rspec-support.git"
-gem "sqlite3", "~> 1.3", "< 1.4", platform: %i[ruby mswin mingw x64_mingw]
-gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw]
+gem "rubocop", "~> 0.66"
```

Comment:
Rubocop produces warnings for non-alphabetical sorted gems AFAIR

Reply:
I think we should let Rubocop ignores this dir

Reply:
Ah, it's Appraisal. Make sense. Could you please add it?

Reply:
I already done this <https://github.com/doorkeeper-gem/doorkeeper/blob/master/.rubocop.yml#L7>